### DISCUSSION TITLE
v2.10.0--c44a180

### DISCUSSION THREAD

**Discussion by Terracotta-6** [artisan-roaster-scope/artisan#1325] [General]
Use the Designer ROR axis change.
[artisan-settings.aset.txt](https://github.com/artisan-roaster-scope/artisan/files/13520765/artisan-settings.aset.txt)
[G1 batch 1.alog.txt](https://github.com/artisan-roaster-scope/artisan/files/13520766/G1.batch.1.alog.txt)

**Comment by Terracotta-6** [comment]
![20231201124707](https://github.com/artisan-roaster-scope/artisan/assets/149906183/ddcc7601-1a14-430d-8a23-6ef946f584e4)
![20231201124732](https://github.com/artisan-roaster-scope/artisan/assets/149906183/33f0aaea-942e-4715-a198-3dd97e523e25)

**Comment by Terracotta-6** [comment]
https://github.com/artisan-roaster-scope/artisan/assets/149906183/bea30282-4382-49e8-a0a8-a2da7e7ad11f

**Comment by MAKOMO** [comment]
Thanks for your detailed report. This is fixed and will be part of the next version.
